<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraParametersDialog</class>
 <widget class="QDialog" name="CameraParametersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Camera Calibration Parameters</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_distortion">
     <property name="title">
      <string>Lens Distortion Correction</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_distortion">
        <item>
         <widget class="QLabel" name="label_distortion">
          <property name="text">
           <string>Distortion Order:</string>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_distortion">
          <item>
           <property name="text">
            <string>0 - None (No distortion correction)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1 - First Order (Linear distortion)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2 - Second Order (Quadratic distortion)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3 - Third Order (Cubic distortion)</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_projection">
     <property name="title">
      <string>Camera Projection Model</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_projection">
        <item>
         <widget class="QLabel" name="label_projection">
          <property name="text">
           <string>Projection Type:</string>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_projection">
          <item>
           <property name="text">
            <string>rectilinear</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>equirectangular</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>stereographic</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_projection_help">
        <property name="text">
         <string>Rectilinear: Standard perspective camera
Equirectangular: 360Â° panoramic images  
Stereographic: Wide-angle fisheye lens</string>
        </property>
        <property name="styleSheet">
         <string>color: gray; font-size: 9pt;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_optimization">
     <property name="title">
      <string>Optimization Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="checkBox_centers">
        <property name="text">
         <string>Optimize camera center position</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_separate_xy">
        <property name="text">
         <string>Use separate focal lengths for X and Y axes</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_buttons">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_reset">
       <property name="text">
        <string>Reset to Defaults</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ok">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_ok</sender>
   <signal>clicked()</signal>
   <receiver>CameraParametersDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>CameraParametersDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>